Systems/c172p-damage.xml

```diff
@@ -288,20 +289,14 @@
             <test logic="AND" value="0.12">
                 /velocities/airspeed-kt GT damage/limits-vne-low
                 damage/roll-moment GT 4000.0
-                wing-damage/left-wing EQ 0.0
+                wing-damage/left-wing LT 0.12
             </test>
 
             <!-- Over-g -->
             <test logic="AND" value="0.12">
-                damage/force-aero-lbs GT damage/limits-lift-medium
```

Comment:
Here you removed a whole block it seems. What the original code says is that the wing gets light damage if the force > low limit and you roll to the right very fast (`roll-moment GT 4000`). Or if the force > medium limit and and the roll moment is between -4000.0 and 4000.0.

So in other words, if the force > low limit, then you must roll very fast, but if force > medium limit, then you can get damage even if you roll slowly.